htdocs/admin/dict.php

```diff
@@ -2620,7 +2627,7 @@
 }
 
 print '<br>';
-
+print dol_get_fiche_end();
```

Comment:
There is  test on line 1265 to execute dol_get_fiche_head so we shuld have same test to do dol_get_fiche_end().
Otherwise code is not correclty balanced.

Reply:
yes, i forgot it